docs/4.x/generated/Piwik/Plugin/Controller.md

```diff
@@ -152,7 +152,7 @@ If the date is absolute, ie. YYYY-MM-DD, it will not be converted to the timezon
        `'today'`, `'yesterday'`, `'YYYY-MM-DD'`
     - `$timezone` (`string`) &mdash;
        The timezone to use.
-- It returns a [`Date`](../../Piwik/Date.md) value.
+- It returns a `Piwik\Plugin\Date` value.
```

Comment:
Here it also should be still `Piwik\Date` as `Piwik\Plugin\Date` doesn't exist and it should link to the page. Same above for `Piwik\Plugin\Site` which should be `Piwik\Site`